jira/database/database.go

```diff
@@ -86,6 +87,8 @@ type Database interface {
 	CreateUser(ctx context.Context, u *User) error
 	GetUser(ctx context.Context, accID AccID, id UserID) (*User, error)
 	ListUsers(ctx context.Context, accID AccID) (UserIter, error)
+	CreateUserWorkaround(ctx context.Context, uw *UserWorkaround) error
```

Comment:
```suggestion
	CreateUserWorkaround(ctx context.Context, accID AccID, workaround bool) error
```
Not worth a separate struct.

Reply:
Done.